src/doc/style-guide/src/expressions.md

```diff
@@ -400,7 +400,12 @@ constructs. For example, a macro use `foo!(a, b, c)` can be parsed like a
 function call (ignoring the `!`), so format it using the rules for function
 calls.
 
-### Special case macros
+The style guide defines specific formatting for particular macros in the
+language or standard library. The style guide does not define formatting for
+any third-party macros, even if similar to those in the language or standard
```

Comment:
Not to be a pedant but would it worth capturing what specifically we mean by "similar", or is it intentionally semi-open to interpretation?

Asking based on some of the hypotheticals that were asked in zulip and the alias/overlapping name scenario or if it's just saying that it doesn't matter/apply even if the 3rd party args/tokens are the same?

Reply:
I meant it to be fairly open-ended; an example being that we comment on format-string-based macros, but don't define styling for the many format-string-based macros outside the standard library.

No objections to adding specifics, though. But I think commenting on the name-based identification of macros would be adding rustfmt-specific bits in the style guide?

Reply:
Yeah that's fair push back, thanks!